.github/workflows/nightly.yml

```diff
@@ -61,3 +61,27 @@ jobs:
         env:
           ORG_GRADLE_PROJECT_apacheUsername: ${{ secrets.NEXUS_USER }}
           ORG_GRADLE_PROJECT_apachePassword: ${{ secrets.NEXUS_PW }} 
+      - name: Publish Server Docker nightly image
+        run: |
+          ./gradlew \
+            :polaris-server:assemble \
+            :polaris-server:quarkusAppPartsBuild --rerun \
+            -Dquarkus.container-image.build=true \
+            -Dquarkus.container-image.push=true \
+            -Dquarkus.container-image.name=polaris \
+            -Dquarkus.container-image.tag=$(date +%Y%m%d) \
+            -Dquarkus.container-image.additional-tags="" \
```

Comment:
Nit: you could use `latest` here (and below)

Reply:
I'm not fan at all of a separate repo for nightly/snapshots.
I think it's better for user to just use the same repo with a specific tag format.

Reply:
I'm ok to use nightly instead of date.

Reply:
I updated to use `nightly` as tag/version.

Reply:
Reading Renovate's [doc](https://docs.renovatebot.com/docker), it seems it is smart enough to never update a `x.y.z` tag to a `nightly` tag:

> If the image tag in use "looks" like a version (e.g. myimage:1, myimage:1.1, myimage:1.1.0, myimage:1-onbuild) then Renovate checks the Docker registry for upgrades (e.g. from myimage:1.1.0 to myimage:1.2.0)

The above seems to imply that the `nightly` tag would be ignored since it doesn't "look like" a version. 

But I didn't verify if that's indeed the case.